tests/integration/conftest.py

```diff
@@ -227,8 +227,10 @@ def fixture_consumer(
         # Speed things up for consumer tests to discover topics, etc.
         topic_metadata_refresh_interval_ms=200,
     )
-    yield consumer
-    consumer.close()
+    try:
+        yield consumer
+    finally:
+        consumer.close()
```

Comment:
Just noting that this shouldn't have any impact, fixtures do not work like a context manager, and the pytest framework will resume the generator on clean-up. So the `finally:` here doesn't make any difference (but also doesn't hurt).